api/swagger.yml

```diff
@@ -5159,6 +5159,66 @@ paths:
         default:
           $ref: "#/components/responses/ServerError"
 
+  /repositories/{repository}/metadata/object/{type}/{object_id}:
+    parameters:
+      - in: path
+        name: repository
+        required: true
+        schema:
+          type: string
+      - in: path
+        name: object_id
+        required: true
+        schema:
+          type: string
+      - in: path
+        name: type
+        required: true
+        schema:
+          type: string
+          enum:
+            - range
+            - meta_range
+    get:
+      tags:
+        - internal
+      operationId: getMetadataObject
+      summary: return a lakeFS metadata object by ID
+      parameters:
```

Comment:
? should add Range header?

Reply:
I don't know if rocksdb (well, pebble) sstables could be partially read - we certainly didn't implement such a reader. What would be the use case?